README.md

```diff
@@ -218,6 +218,25 @@ locally. To serve a production build locally:
     attempt to run the build on the same port specified in the
     `env.config.js` file.
 
+## Local module configuration for TypeScript
+
+#. Create file in repository `tsconfig.json`, with a clause `"extends": "@openedx/frontend-build"`
+#. Set "rootDir" to the root of the source code folders
+#. Set "include" to wildcard patterns specifying the subdirectories/files under rootDir where source code can be found
+#. Include any wildcards under rootDir that should be excluded using "exclude"
+
+```Sample json
+{
+  "extends": "@openedx/frontend-build",
```

Comment:
It looks like we're actually extending `@edx/typescript-config` in the tsconfig in this repo.  Should we not recommend doing the same elsewhere?

Reply:
Also, it might be good to recommend setting `outDir`, because I found in https://github.com/openedx/paragon/pull/3016 that if you extend `@edx/typescript-config` but don't set `outDir`, when you run `tsc` the files will get output to `./node_modules/@edx/typescript-config/dist/` - which is not that different from sending them to `/dev/null`. Most people won't be running `tsc` (rather webpack, babel, etc.) but if they do this heads up will save them some time trying to figure out why the output is not showing up.

Reply:
> It looks like we're actually extending @edx/typescript-config in the tsconfig in this repo. Should we not recommend doing the same elsewhere?

@arbrandes I believe the original intent for recommending `@openedx/frontend-build` here was that there might be MFE-specific additions/updates we want in the base TS config available to MFEs that don't make sense in the upstream `@edx/typescript-config` (which, in theory, could be used for more beyond just MFEs).

For example, the base ESLint config provided to MFEs via frontend-build extends `@edx/eslint-config`, but slightly differs. This approach to extend `@openedx/frontend-build` was largely following that paradigm.

That said, if supporting MFE-only TypeScript config isn't necessary (which is probably isn't; I don't really see legacy UI repos like edx-platform adopting TypeScript anytime soon), I agree recommending to extend `@edx/typescript-config` directly makes more sense.

Reply:
Ah, I've always wondered why `typescript-config` and `eslint-config` existed. Personally, I'd prefer to just consolidate them into `frontend-build` or `frontend-platform`.